### PR TITLE
added unit test for accuracy

### PR DESCRIPTION
**Pull request by jclagarde**

### CODE REVIEW

File: lineage/tests/test_tHMM.py [hunk 1]
```diff
@@ -102,3 +102,17 @@ def test_get_leaf_NF(self):
...
+        Really defined states should get an accuracy >95%. 
+        Lineages used should be large and distinct.
+        """
+        X = [LineageTree.init_from_parameters(pi, T, E, (2**11))]
+        tree = tHMM(X, 2)
+        tree.fit()
+        predicted_states = tree.predict()
+        true_states = [cell.state for lineage in X for cell in lineage.output_lineage]
+        accuracy = sum([1 if i == j else 0 for i,j in zip(predicted_states[0], true_states)])/len(predicted_states)
+        self.assertGreaterEqual (accuracy, 95)
```
Comment (shak360) [inline lineage/tests/test_tHMM.py:117]:
```suggestion
        self.assertGreaterEqual(accuracy, 95)
```

File: lineage/tests/test_tHMM.py [hunk 2]
```diff
@@ -102,3 +102,17 @@ def test_get_leaf_NF(self):
...
+        """
+        Really defined states should get an accuracy >95%. 
+        Lineages used should be large and distinct.
+        """
+        X = [LineageTree.init_from_parameters(pi, T, E, (2**11))]
+        tree = tHMM(X, 2)
+        tree.fit()
+        predicted_states = tree.predict()
+        true_states = [cell.state for lineage in X for cell in lineage.output_lineage]
+        accuracy = sum([1 if i == j else 0 for i,j in zip(predicted_states[0], true_states)])/len(predicted_states)
```
Comment (shak360) [inline lineage/tests/test_tHMM.py:116]:
```suggestion
        accuracy = sum([1 if i == j else 0 for i,j in zip(predicted_states[0], true_states)])/len(predicted_states[0])
```

File: lineage/tests/test_tHMM.py [hunk 3]
```diff
@@ -102,3 +102,17 @@ def test_get_leaf_NF(self):
         for ind, NFlin in enumerate(NF):
             self.assertGreaterEqual(NFlin.shape[0], 0)  # at least zero cells in each lineage
             self.assertGreaterEqual(NF3[ind].shape[0], 0)
+            
+    def test_level_of_performance(self):
+        """
+        Really defined states should get an accuracy >95%. 
+        Lineages used should be large and distinct.
+        """
+        X = [LineageTree.init_from_parameters(pi, T, E, (2**11))]
```
Comment (shak360) [inline lineage/tests/test_tHMM.py:111]: You need to define `pi`, `T`, and `E`. Either use the ones defined in `self`, that is `self.pi`, `self.T`, and `self.E` or define them inside this funciton.

**Comment by jclagarde** [comment]
> Can you use the accuracy function used in the figures? I'd like to have at least something run that.

@aarmey I updated the branch to use the accuracy function Analyze and Results from analyze.py. When you get the chance, please take a look at it

File: lineage/tests/test_tHMM.py [hunk 4]
```diff
@@ -10,6 +10,8 @@
 from ..tHMM import tHMM
 from ..states.StateDistributionGaPhs import StateDistribution as StateDistPhase
 from ..figures.figureCommon import pi, T, E
+from ..Analyze import Analyze
+from ..Analyze import Results
```
Comment (shak360) [inline lineage/tests/test_tHMM.py:14]:
```suggestion
from ..Analyze import Analyze, Results
```

**Comment by jclagarde** [comment]
> autopep8 -i -a -r --max-line-length 200 lineage

@shak360 forsure, just did it

**Pull request merged**